### PR TITLE
Changes to make it easier for downstream images to override…

### PR DESCRIPTION
**Pull request by zmoore-usgs**
… launch-app.sh

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -25,6 +25,7 @@ ENV serverContextPath=/
 ENV keystoreLocation=$HOME/localkeystore.pkcs12
 ENV keystorePassword=changeme
 ENV keystoreSSLKey=tomcat
+ENV LAUNCH_APP_SCRIPT=$HOME/launch-app.sh
```
Comment (zmoore-usgs) [inline Dockerfile]: I moved this to an ENV so downstream images can access it and use `COPY` or `ADD` to put their script in the right spot.

File: Dockerfile [hunk 2]
```diff
@@ -33,14 +34,13 @@ RUN apt-get update && \
...
-RUN [ "chmod", "+x", "pull-from-artifactory.sh", "entrypoint.sh", "launch-app.sh" ]
-RUN chown $USER:$USER pull-from-artifactory.sh entrypoint.sh launch-app.sh
+COPY --chown=1000:1000 pull-from-artifactory.sh pull-from-artifactory.sh
+COPY --chown=1000:1000 entrypoint.sh entrypoint.sh
+COPY --chown=1000:1000 launch-app.sh $LAUNCH_APP_SCRIPT
+RUN chmod +x pull-from-artifactory.sh entrypoint.sh $LAUNCH_APP_SCRIPT
 USER $USER
 
-RUN ./pull-from-artifactory.sh wma-maven-centralized gov.usgs.wma spring-boot-sample ${artifact_version} app.jar
+RUN ./pull-from-artifactory.sh wma-maven-centralized gov.usgs.wma spring-boot-sample $artifact_version app.jar
```
Comment (zmoore-usgs) [inline Dockerfile]: This was just done for consistency

File: Dockerfile (hunk 1 above)
Comment (isuftin) [reply Dockerfile]: Makes sense but why only have this one script in the $HOME location and not the others or vice versa?

Comment (isuftin) [reply Dockerfile]: I'd say this is good enough for now. Experience with working with these images will dictate to us whether or not we should change it up

**Pull request merged**